src/rhc.erl

```diff
@@ -927,6 +953,26 @@ erlify_server_info(<<"nodename">>, Name) -> {node, Name};
 erlify_server_info(<<"riak_kv_version">>, Vsn) -> {server_version, Vsn};
 erlify_server_info(_Ignore, _) -> [].
 
+%% @doc Convert a preflist resource response to a proplist.
+erlify_preflist(Response) ->
+    Preflist = [V || {_, V} <- proplists:get_value(<<"preflist">>, Response)],
+    lists:foldl(
```

Comment:
This can be simplified to a list comprehension.

```erlang
[ lists:foldl(fun({K, V}, AccRec) -> erlify_preflist(K, V, AccRec) end, #preflist_item{}, P) || P <- Preflist ]
```